install-and-launch.sh

```diff
@@ -62,15 +73,22 @@ if [ "$interactive" == "true" ] && [ -z "$sublime_host" ]; then
     read -rp "Please specify where your Sublime is deployed (default: localhost): " sublime_host </dev/tty
 fi
 
-echo "Cloning Sublime Platform repo"
-if ! git clone https://github.com/sublime-security/sublime-platform.git; then
-  echo "Failed to clone Sublime Platform repo"
-  exit 1
+if [ -z "$clone_platform" ]; then
+    clone_platform=true
 fi
 
-echo "Launching Sublime Platform"
-cd sublime-platform || { echo "Failed to cd into sublime-platform"; exit 1; }
+if [ "$clone_platform" == "true" ]; then
+    echo "Cloning Sublime Platform repo"
+    if ! git clone https://github.com/sublime-security/sublime-platform.git; then
```

Comment:
quick nit: `--depth=1` to make this shallow

Reply:
That's a great call out! Done!